Command/GenerateDoctrineCrudCommand.php

```diff
@@ -45,7 +45,7 @@ protected function configure()
                 new InputOption('entity', '', InputOption::VALUE_REQUIRED, 'The entity class name to initialize (shortcut notation)'),
                 new InputOption('route-prefix', '', InputOption::VALUE_REQUIRED, 'The route prefix'),
                 new InputOption('with-write', '', InputOption::VALUE_NONE, 'Whether or not to generate create, new and delete actions'),
-                new InputOption('format', '', InputOption::VALUE_REQUIRED, 'Use the format for configuration files (php, xml, yml, or annotation)', 'annotation'),
+                new InputOption('format', '', InputOption::VALUE_REQUIRED, 'Use the format for routing (php, xml, yml, or annotation)', 'annotation'),
```

Comment:
Yes you are right. I say maiby a stupidity but changing the name of an option is not considered as BC break ?